oaknational/Oak-Web-Application
# Feat/pupil 892/quiz result question stem

## Description

Music year: 1976

- List of changes
created QuizResultsQuestionStem component and story

and added a story to quizquestionstem (just because)

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2651--owa-storybook.netlify.thenational.academy/?path=/story/components-pupilcomponents-quizresultquestionstem--default
2. Click on default/ text and image 
3. You should see the question stem variations

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
